### PR TITLE
chore: update header order, components list and links

### PR DESCRIPTION
**Pull request by Knorgias**
## What I did

1. Moved Docs content to -> (new) section Fundamentals
2. Removed Blog section from header
3. Moved all components one level higher, no separation on type of component anymore
4. Reordered the header order
5. Fixed broken links in codebase

### CODE REVIEW

File: docs/components/inputs/input/overview.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const main = () => html`<lion-input label="First Name"></lion-input>`;
   - **before**: does not have an active use case yet, but the option is in place.
   - **after**: can be used to show a currency or percentage.
 - Delegates attributes like `type`, `disabled`, `placeholder` and `readonly` to the native input.
-- Can make use of [validation](../../docs/../../docs/systems/form/validate.md).
+- Can make use of [validation](../../docs/../../fundamentals/systems/form/validate.md).
```
Comment (gerjanvangeest) [inline docs/components/inputs/input/overview.md:26]:
```suggestion
- Can make use of [validation](../../../fundamentals/systems/form/validate.md).
```

File: docs/fundamentals/index.md [hunk 2]
```diff
@@ -11,9 +11,11 @@ Let's see what each section describes.
 
 1. [Systems](./systems/core/overview.md) <br>
    These are the fundamental systems that power all of our components.
-2. [Tools](./tools/helpers/overview.md) <br>
+2. [Principles](../guides/principles/index.md) <br>
```
Comment (gerjanvangeest) [inline docs/fundamentals/index.md:14]: This should be removed from this overview again

File: docs/guides/index.md [hunk 3]
```diff
@@ -23,11 +23,11 @@ Developers extending our web components, for instance: `class MaterialInput exte
...
 - [Subclasser APIs](./principles/subclasser-apis.md)
-- [Extend documentation](../blog/extending-documentation/)
+- [Extend documentation](../blog/extending-documentation.md)
 
 ## How To
 
 Next to that we have some specific "How To" documentations.
 
-- [Get started](./how-to/get-started.md)
+- [Get started](./how-to/get-started/index.md)
```
Comment (gerjanvangeest) [inline docs/guides/index.md:32]: This should stay `get-started.md` i believe

File: docs/guides/principles/index.md [hunk 4]
```diff
@@ -1 +1,26 @@
-# Principles ||10
+# Principles ||30
```
Comment (gerjanvangeest) [inline docs/guides/principles/index.md:1]: Keep "Principles" above "How To", feels more right.

File: docs/components/button/overview.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Interaction >> Button >> Overview ||10
```
[the remark this answers is not in the document]
Comment (gerjanvangeest) [reply docs/components/button/overview.md:1]: Maybe  store them for now as keyword in the package.json?

[the remark this answers is not in the document]
Comment (tlouisse) [reply docs/components/button/overview.md:1]: Yeah could be. Or frontmatter... What do you think @daKmoR ?

[the remark this answers is not in the document]
Comment (daKmoR) [reply docs/components/button/overview.md:1]: let's not complicate things - if we need categories we can add them back later

[the remark this answers is not in the document]
Comment (tlouisse) [reply docs/components/button/overview.md:1]: Yeah agreed, we can add them again later. I like the fact that we can add multiple categories now, allows you to think in a more flexible way about components (like https://html.spec.whatwg.org/multipage/dom.html#kinds-of-content)

**Comment by tlouisse** [comment]
Q: (why) do we want to remove blogs on Lion?

**Review by tlouisse — APPROVED** [review]

**Pull request merged**